pyodbc.pyi,sha256=JWNpdgmrxhNioTjW6OoAEcXELcGZobcMYzKtYlcJRuY,32320
pyodbc.cpython-36m-x86_64-linux-gnu.so,sha256=UF4U7Q4k2KTdYwvJlah-6aA2-t0YyKX6D9RU94NPHKQ,907568
pyodbc-4.0.39.dist-info/WHEEL,sha256=YVAXCacQXnznQ-_lMj8Nvqrs7YPtMP1pHRCw9EcyWRQ,150
pyodbc-4.0.39.dist-info/LICENSE.txt,sha256=mFnIV3DJG3hmsPdpyTjjbWTQJgFBtSm5R7Tq2WiD6rQ,868
pyodbc-4.0.39.dist-info/RECORD,,
pyodbc-4.0.39.dist-info/METADATA,sha256=dg4u-2uCApYwzVzr6412dJ_r6q24s9fvIi3uWEnubi8,3180
pyodbc-4.0.39.dist-info/top_level.txt,sha256=Rb69vS-SNN_Bfh99vX7eCcvEAJoyzowG3u4R7W36Flw,8
